features/steps/steps.rb

```diff
@@ -238,19 +238,44 @@ def get_version_number(file_path)
 
 # React Native
 Before('@BuildRNAndroid') do
-  unless defined?(@setup_android) && @setup_android
+  unless defined?($setup_android) && $setup_android
     puts "Setting up React Native Android app and sourcemap..."
     @output = `node features/react-native/scripts/generate.js`
     Maze.check.include(`ls features/react-native/fixtures/generated/old-arch/#{ENV['RN_VERSION']}/android/app/build/generated/sourcemaps/react/release`, 'index.android.bundle.map')
-    @setup_android = true
+
+    if ENV['RN_VERSION'].to_f == 0.70
+      ENV['APP_MANIFEST_PATH'] = "features/react-native/fixtures/generated/old-arch/#{ENV['RN_VERSION']}/android/app/build/intermediates/merged_manifests/release/AndroidManifest.xml"
+      ENV['BUNDLE_PATH'] = "features/react-native/fixtures/generated/old-arch/#{ENV['RN_VERSION']}/android/app/build/generated/assets/react/release/index.android.bundle"
+      ENV['SOURCE_MAP_PATH'] = "features/react-native/fixtures/generated/old-arch/#{ENV['RN_VERSION']}/android/app/build/generated/sourcemaps/react/release/index.android.bundle.map"
+    elsif ENV['RN_VERSION'].to_f == 0.73
+      ENV['APP_MANIFEST_PATH'] = "features/react-native/fixtures/generated/old-arch/#{ENV['RN_VERSION']}/android/app/build/intermediates/merged_manifests/release/AndroidManifest.xml"
+      ENV['BUNDLE_PATH'] = "features/react-native/fixtures/generated/old-arch/#{ENV['RN_VERSION']}/android/app/build/generated/assets/createBundleReleaseJsAndAssets/index.android.bundle"
+      ENV['SOURCE_MAP_PATH'] = "features/react-native/fixtures/generated/old-arch/#{ENV['RN_VERSION']}/android/app/build/generated/sourcemaps/react/release/index.android.bundle.map"
+    elsif ENV['RN_VERSION'].to_f == 0.75
+      ENV['APP_MANIFEST_PATH'] = "features/react-native/fixtures/generated/old-arch/#{ENV['RN_VERSION']}/android/app/build/intermediates/merged_manifests/release/processReleaseManifest/AndroidManifest.xml"
+      ENV['BUNDLE_PATH'] = "features/react-native/fixtures/generated/old-arch/#{ENV['RN_VERSION']}/android/app/build/generated/assets/createBundleReleaseJsAndAssets/index.android.bundle"
+      ENV['SOURCE_MAP_PATH'] = "features/react-native/fixtures/generated/old-arch/#{ENV['RN_VERSION']}/android/app/build/generated/sourcemaps/react/release/index.android.bundle.map"
+    end
```

Comment:
How about expressing it like this to make it clearer what the differences are? (It pushes the full feature path to the feature script and allows this to be just the differences between versions - so we can add new ones if the path changes again.)

```suggestion
    ENV['APP_MANIFEST_PATH'] = "android/app/build/intermediates/merged_manifests/release/processReleaseManifest/AndroidManifest.xml"
    ENV['BUNDLE_PATH'] = "android/app/build/generated/assets/createBundleReleaseJsAndAssets/index.android.bundle"
    ENV['SOURCE_MAP_PATH'] = "android/app/build/generated/sourcemaps/react/release/index.android.bundle.map"
    if ENV['RN_VERSION'].to_f < 0.75
      ENV['APP_MANIFEST_PATH'] = "android/app/build/intermediates/merged_manifests/release/AndroidManifest.xml"
    elsif ENV['RN_VERSION'].to_f < 0.73
      ENV['BUNDLE_PATH'] = "android/app/build/generated/assets/react/release/index.android.bundle"
```